src/newgrp.c

```diff
@@ -390,6 +390,9 @@ int main (int argc, char **argv)
 #ifdef WITH_AUDIT
 	audit_help_open ();
 #endif
+
+	check_fds ();
+
```

Comment:
Could you document why some programs don't sanitize_env()?

Reply:
I'm not sure why some programs don't call sanitize_env. When I tried adding sanitize_env to the programs that were missing it, tests failed. You can see the tests failing [here](https://github.com/skyler-ferrante/shadow/actions/runs/8207201528/job/22447995357). We could add sanitize_env to other programs but I think that should probably go into another pull request.

Reply:
Makes sense.